### ISSUE TITLE
Hidden events

### ISSUE THREAD

**Issue by parkkot** [hackjunction/hackplatform#62]
Ability to create hidden events not visible on the platform publically, only through a link/code etc.

**Comment by lappalj4** [comment]
Good idea. Do you feel it would be more invite-only as in you need some sort of code to access the event, or just not visible on the frontpage but accessible via the direct link to i.e. /events/junction-2019?

**Comment by parkkot** [comment]
The second option is a lot easier to implement so that could serve the purpose for now. Maybe in the longer term, adding a code required -feature would be good.
